roles/confluent.common/tasks/remove_packages.yml

```diff
@@ -26,11 +26,19 @@
     msg: "{{ confluent_packages_removed }}"
   when: confluent_packages_removed|length > 0
 
+### Kafka Connect Replicator Binary does not include a service definition by default, thus we need to validate if one exists in order to handle service lifecycle correctly.
+### - ansible_facts.services[service_name + '.service'] is defined
```

Comment:
is this second line needed?

Reply:
Yes it is.

Reply:
Care to elaborate? looks like a duplicate of line 40